tf2onnx/onnx_opset/nn.py

```diff
@@ -2091,3 +2091,81 @@ def version_11(cls, ctx, node, **kwargs):
         ctx.replace_all_inputs(node.output[3], sum_max_neg)
 
         ctx.remove_node(node.name)
+
+
+@tf_op("ExtractImagePatches")
+class ExtractImagePatches:
+    @classmethod
+    def version_9(cls, ctx, node, **kwargs):
+        input_shape = ctx.get_shape(node.input[0])
+        output_shape = node.output_shapes[0]
+
+        sizes = node.get_attr_value("ksizes")
+        strides = node.get_attr_value("strides")
+        rates = node.get_attr_value("rates")
+        padding = node.get_attr_str("padding")
+
+        # Our constraints.
```

Comment:
Could you please provide more details about this constraint so people know how to improve in future?

Reply:
I've expanded this comment and given an example of a call that succeeds in Tensorflow but fails for this particular implementation.